src/main/java/net/wurstclient/commands/DistCmd.java

```diff
@@ -0,0 +1,94 @@
+package net.wurstclient.commands;
+
+import net.minecraft.util.math.BlockPos;
+import net.wurstclient.command.CmdException;
+import net.wurstclient.util.ChatUtils;
+import net.wurstclient.command.CmdError;
+import net.wurstclient.command.CmdException;
+import net.wurstclient.command.CmdSyntaxError;
+import net.wurstclient.command.Command;
+import java.lang.Math;
+
+
+public class DistCmd extends Command {
+
+
+    public DistCmd() {
+        super("dist", "Gets Distance between 2 Cordinates\nYou can convert Overworld Cords To Nether cords With cvt:<x> <z>\n ", ".dist <x> <z>:<x> <z>", ".dist cvt:<x> <z>");
+    }
+
+    @Override
+    public void call(String[] args) throws CmdException {
+
+        if(args.length < 1)
+            throw new CmdError("Enter Command Properly");
+
+        String Allcords = String.join(" ", args);
+        if(!Allcords.contains(":"))
+            throw new CmdError("Separate Cordinates With : ");
+
+        if (Allcords.startsWith("cvt")){
+
+            String[] arrOfSTR = Allcords.split(":", 0);
+            String cordsWithSpace = arrOfSTR[1];
+            String[] Cordsarray = cordsWithSpace.split(" ",0);
+            if (Cordsarray.length < 2)
+                throw new CmdError("Provide Cordinates Properly");

```

Comment:
We can already assume that args positions 1 and 2 will contain the coordinates we are looking for. There's no need to create more local variables to redefine info already easily accessible to us.

Instead we can check the length of the args array.